### ISSUE TITLE
JUNC_DIST and JUNC_TYPE in Roads_emission_from_traffic

### ISSUE THREAD

**Issue by danigras** [Universite-Gustave-Eiffel/NoiseModelling#555]
HI,
 I want to simulate a traffic light intersection and I have to indicate the distance from this intersection. Looking at the regulation, I think a point shape (traffic point) is needed and the software must calculate the factor based on the distance from each point (the road is divided into point sources). 
Paragraph 2.2.5. https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CELEX:32015L0996&from=PT 
On the other hand, it seems limiting to me to enter only the distance from the intersection
I hope I made myself clear.
Daniele

**Comment by pierromond** [comment]
Yes, this is a critic point. The best practice and the one shared by lot of software, is it to cut your road segment near the intersection in 10 m long segment. Each one of this segment have a distance value equal to the distance between the centroid and the section. You can do this with some Spatial SQL function. More than a bug, this is sthg aht we could implement in the software. A WPS block that prepare the road layer spliting it in small segments near intersection... Maybe in the future !

> On the other hand, it seems limiting to me to enter only the distance from the intersection 
You have distance, and intersection type... This is CNOSSOS-EU. But you can implement in NM the acceleration parameter also.

**Comment by pierromond** [comment]
related to #524
